### PR TITLE
Create SECURITY.md file

### PR DESCRIPTION
**Pull request by Stratus3D**
# Summary

Create SECURITY.md file with list of supported versions and secure contact information for reporting vulnerabilities.

Relates to: #651

## Other Information

@asdf-vm/core - if you have secure method of being contacted and want to handle vulnerability reporting please add your username and contact address to the list. It doesn't have to be email. I still do not have PGP setup for my own email address yet.

### CODE REVIEW

**Comment by danhper** [comment]
Added my info

**Comment by Stratus3D** [comment]
Thanks @danhper ! can you recommend any good resources on setting up PGP for email? I use GPG and pass for password management, but I've not yet done anything with PGP.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Security Policy
+
+## Supported Versions
+
+Typically only the latest version is maintained. Versions are
+released sequencially, with major, minor, and patch versions incremented
```
Comment (jthegedus) [inline SECURITY.md]: sequentially

**Comment by jthegedus** [comment]
Should we specify a minimum time for us to respond and address the issue before the reporter should make a public disclosure?

**Comment by Stratus3D** [comment]
>    1. Should we specify a minimum time for us to respond and address the issue before the reporter should make a public disclosure?

I always try to respond to important issues as fast as possible, but my response times vary. I'm not willing to commit to addressing security issues within a set time frame. I will give them priority, but I'm not willing to commit to anything more.

>    2. I think we should explicitly call out that this only applies to the asdf-vm/asdf core tool and not any plugins whose authors are responsible for their own security policies and guarantees. Thoughts?

Yes, totally agree. I will update the PR.

**Comment by Stratus3D** [comment]
I'm using my own email server and Thunderbird is my email client, it shouldn't be too hard for me to setup, I just have to take the time to learn how to do it.

**Pull request merged**